### DISCUSSION TITLE
client9/misspell references

### DISCUSSION THREAD

**Discussion by ccoVeille** [golangci/golangci-lint#4658] [General] [answered]
The repository https://github.com/client9/misspell is unmaintained.

You made a fork and maitain it https://github.com/golangci/misspell

Shouldn't these files refers to the new repositor ?

https://github.com/golangci/golangci-lint/blob/003b04806989db88a148f392b7edb00eb95833a2/assets/linters-info.json#L895
https://github.com/golangci/golangci-lint/blob/003b04806989db88a148f392b7edb00eb95833a2/pkg/lint/lintersdb/builder_linter.go#L552

I was suprised to see the link in the documentation was pointing on client9 code.

While I understand, it's legitimate to mention the first author, the code current ran is now maintained by you

https://github.com/golangci/golangci-lint/blob/003b04806989db88a148f392b7edb00eb95833a2/pkg/golinters/misspell/misspell.go#L3-L11

**Comment by ldez** [comment, marked as answer]
We use a fork but the original author and project are still the same.
Those links are used to reference the [linter's authors](https://golangci-lint.run/product/thanks/).

**Reply by ccoVeille** [reply]
I know, but I would have expected the link on github link of the linter to be the one of your repository on this page

https://golangci-lint.run/usage/linters/#disabled-by-default

I raised my point, I understand your point of view
